cmd/kube-rbac-proxy/app/kube-rbac-proxy.go

```diff
@@ -218,6 +218,8 @@ func Run(cfg *completedProxyRunOptions) error {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	klog.SetLogFilter(&SanitizingFilter{})
```

Comment:
I wouldn't put it into the `Run` function. IRather in the `NewKubeRBACProxyCommand`. Maybe in `PersistentPreRunE`.